### PR TITLE
revenant harvest now logs properly

### LINKED ISSUE #65463: Ghost in the machine/How revenants are not logged properly
## Reproduction:

- Have a revenant on station.
- Stuff happens
- You kinda need logs
- Look at logs
- Notice that only Overload Lights/Defile/Blight/Transmit are apparently logged
- Cry because you need the Harvest interaction logged
- Get banned

### PR DESCRIPTION
**Pull request by capsaicinz**
## About The Pull Request

harvesting as a revenant used to give no logging. now it does, surprise!

fixes #65463

## Why It's Good For The Game

good for combat logs and stuff.

## Changelog
:cl:
fix: revenant harvest now logs properly
/:cl:

### CODE REVIEW

**Comment by lewcc** [comment]
While you're at it, might want to add a log message for when a rev kills by harvesting an unconscious soul?

**Comment by capsaicinz** [comment]
> While you're at it, might want to add a log message for when a rev kills by harvesting an unconscious soul?

good idea

File: code/modules/antagonists/revenant/revenant_abilities.dm [hunk 1]
```diff
@@ -38,6 +38,7 @@
 		if(prob(10))
 			to_chat(target, span_revennotice("You feel as if you are being watched."))
 		return
+	src.log_message(span_warning("has started to harvest [target][(!target.mind) ? "(no mind)" : ""]."), LOG_ATTACK)
```
Comment (AnturK) [inline code/modules/antagonists/revenant/revenant_abilities.dm:41]:
```suggestion
	log_message(span_warning("has started to harvest [key_name(target)]."), LOG_ATTACK)
```
No need for src, and use the key_name() helper

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/revenant/revenant_abilities.dm (hunk 1 above)
Comment (ShizCalev) [reply code/modules/antagonists/revenant/revenant_abilities.dm:41]: should be log_combat so both parties get the entry.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**